api/types/time/timestamp_test.go

```diff
@@ -49,8 +49,7 @@ func TestGetTimestamp(t *testing.T) {
 		{"1.5h", fmt.Sprintf("%d", now.Add(-90*time.Minute).Unix()), false},
 		{"1h30m", fmt.Sprintf("%d", now.Add(-90*time.Minute).Unix()), false},
 
-		// String fallback
-		{"invalid", "invalid", false},
+		{"invalid", "", true},
```

Comment:
`expectedErr` should probably the a string with the expected error instead of just a boolean, but this is existing so doesn't need to be fixed in this PR.